Eastern-Research-Group/mywaterway
# Feature/remove watershed shading wsio

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3810819

## Main Changes:
* Added code to toggle the watershed shading on or off depending upon whether the wsio layer is on or off.
* Added a new visual test for this new feature.

## Steps To Test:
1. Run the `1-visual-test.spec.js` test
2. Verify the new test passes
3. Do a search on the community page
4. Go to the protect tab
5. Verify the huc boundaries have the shading in the middle
6. Turn on the wsio layer
7. Verify the huc boundaries no longer have the shading in the middle
8. Turn the wsio layer off
9. Verify the huc boundaries have the shading in the middle again
